src/databases/OpenPMD/OpenPMDCommonPluginInfo.C

```diff
@@ -11,7 +11,7 @@
 //  Method:  OpenPMDCommonPluginInfo::GetDatabaseType
 //
 //  Purpose:
-//    Returns the type of an openPMD database.
+//    Returns the type of a OpenPMD database.
```

Comment:
I believe you would use "an" here since "OpenPMD" starts with a vowel. I noticed several instances of this in this file and the following 2 files in this list.

Reply:
These were generated via xml2info.

Reply:
Sounds good.